clang/include/clang/CIR/MissingFeatures.h

```diff
@@ -109,6 +109,7 @@ struct MissingFeatures {
   static bool cgFPOptionsRAII() { return false; }
   static bool metaDataNode() { return false; }
   static bool fastMathFlags() { return false; }
+  static bool vlas() { return false; }
```

Comment:
```suggestion
  static bool variableLengthArrays() { return false; }
```

`vla` has a handful of other meanings that this could get confused for.  Also, it gives me an immediate gag-reflex to see `vla` in code, so spelling it out eases me into them better :)